### ISSUE TITLE
DSAI module

### ISSUE THREAD

**Issue by CXJ0612** [zhengshuchen/HCFNet#2]
请问代码里的x_low和x_high与文章中提到的low_level和High_level是反的吗？

**Comment by xxxh111** [comment]
请问一下DASI中的        elif x_low == None:
            x0 = self.conv(torch.cat((x[0], x_high[0]), dim=1))
            x1 = self.conv(torch.cat((x[0], x_high[1]), dim=1))
            x2 = self.conv(torch.cat((x[0], x_high[2]), dim=1))
            x3 = self.conv(torch.cat((x[0], x_high[3]), dim=1))
            这里应该是四个[x0]吗？
